src/jio-footer.ts

```diff
@@ -68,23 +68,21 @@ export class Footer extends LitElement {
    @property({type: String})
    reportAProblemTemplate = "";
 
-   override render() {
-      // Check if the current page is /download or /download/mirrors
-      const isADownloadsPage = this.sourcePath.includes('/download/') || this.sourcePath.includes('/download/mirrors/');
+   @property({ type: Boolean })
```

Comment:
a @property is passed in when you use the html

so <jio-footer skipReportIssue />

That way you don't need the function at all since it doesn't have to guess if its a download page or not.

Reply:
Remove the function entirely
Then render based on !this.skipReportIssue

The url check doesn't belong in the component. It belongs in the site using the component.